schema/2.0.0/corpora_schema.md

```diff
@@ -39,6 +39,12 @@ This document is organized by:
 
 * **AnnData** - The canonical data format for the cellxgene Data Portal is HDF5-backed [AnnData](https://anndata.readthedocs.io/en/latest) as written by version 0.7 of the anndata library.  Part of the rationale for selecting this format is to allow cellxgene to access both the data and metadata within a single file. The schema requirements and definitions for the AnnData `X`, `uns`, `obs`, and `obsm` attributes are described below.
 
+* **Organisms**. Datasets MUST contain either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal. 
```

Comment:
Technically a Dataset could contain Human or Mouse, but also contain other data, so the two statements are somewhat misaligned.
I would suggest "Data MUST be either Human or Mouse data. No other organisms are accepted by the cellxgene Data Portal."
But "Datasets MUST contain either Human or Mouse data." could also imply that a Dataset cannot contain both Human and Mouse data. If that is, indeed, a criterium, then I'd suggest "Datasets MUST contain either only Human or only Mouse data. A Dataset MUST NOT contain both Human and Mouse data, and no other organisms are accepted by the cellxgene Data Portal."

Reply:
We moved organism from `uns` to `obs` to support the mixed dataset. I will update to your suggestion.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/8555f4bdced04471ba2c455e9c08c7ea82ab3d38